db/migrate/20210223150604_add_category_to_characters.rb

```diff
@@ -0,0 +1,20 @@
+class AddCategoryToCharacters < ActiveRecord::Migration[6.1]
+  def up
+    execute <<-SQL
+      CREATE TYPE character_category AS ENUM ('celebrity', 
```

Comment:
We keep the ENUM but let's remember that category could be inserted in a dedicated table for flexibility in particular if we want to let some user create their own categories.